### PR TITLE
Project branch

### PR DESCRIPTION
**Pull request by XeniaSp**

### CODE REVIEW

File: roles/firewall/handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+# handlers file for Firewall
+
+  - name: Restart ufw
+    service: name=ufw state=restarted
```
Comment (mspanakis) [inline roles/firewall/handlers/main.yml]: alakse to structure...den mporw na ta vlepw se mia grammi :P

File: roles/firewall/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      state: present
+
+# Limit connection to certain ports. Ufw will deny connections if an IP
+# address has attempted to initiate 6 or more connections in the last
+# 30 seconds.
+  
+  - ufw:
+      rule: limit
+      port: "{{ http_port }}"
+      proto: tcp
```
Comment (mspanakis) [inline roles/firewall/tasks/main.yml]: check with_items gia to pws na ta kaneis 1 mono task afta...

File: roles/firewall/tests/test.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - Firewall
```
Comment (mspanakis) [inline roles/firewall/tests/test.yml:5]: leei oti thelei new line sto telos

File: roles/intrution/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (mspanakis) [inline roles/intrution/.travis.yml]: intrusion den einai?

File: roles/intrution/handlers/main.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
+# handlers file for intrution
+
+  - name: Restart psad
+    service: name=psad state=restarted
```
Comment (mspanakis) [inline roles/intrution/handlers/main.yml]: structure

File: roles/intrution/meta/main.yml [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+    # List tags for your role here, one per line. A tag is
+    # a keyword that describes and categorizes the role.
+    # Users find roles by searching for tags. Be sure to
+    # remove the '[]' above if you add tags to this list.
+    #
+    # NOTE: A tag is limited to a single word comprised of
+    # alphanumeric characters. Maximum 20 tags per role.
+
+dependencies: []
+  # List your role dependencies here, one per line.
```
Comment (mspanakis) [inline roles/intrution/meta/main.yml]: edw btw mporeis na oriseis dependencies...an px ena role xreiazetai to alo gia na leitourgisei, an to valeis edw, tha ektelestei prwto...opote den xreiazetai na to kaleseis esi prwta

File: roles/intrution/templates/psad.conf [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# Configuration file for Psad Intrusion Detection System.
```
Comment (mspanakis) [inline roles/intrution/templates/psad.conf]: panw panw tha mporouses na valeis kai to ansible_managed variable...Einai kalo practice gia templates gia na ksereis oti afto to arxeio einai generated apo to ansible

File: roles/monitoring/defaults/main.yml [hunk 8]
```diff
@@ -0,0 +1,10 @@
+---
+# defaults file for monitoring
+Email: xeniaspan@gmail.com
```
Comment (mspanakis) [inline roles/monitoring/defaults/main.yml]: edw des an mporeis na to kaneis lookup apo to env...na exeis dld mia env variable setup sto pc sou...gia na min vazeis to email sou edw sto github publicly

File: roles/monitoring/tasks/main.yml [hunk 9]
```diff
@@ -0,0 +1,70 @@
+---
+# tasks file for monitoring
+
+  - name: Install Logwatch
+    apt: 
+      pkg: logwatch
```
Comment (mspanakis) [inline roles/monitoring/tasks/main.yml]: pkg i name?

File: roles/monitoring/tasks/main.yml [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+      state: installed
+
+  - name: Configure Logwatch
+    command: logwatch --detail Medium --mailto {{ Email }} --output mail --service All --range 'between -7 days and today'  
+     
+# Installation of chkrootkit
+
+  - name: Install chkrootkit
+    apt:
+      pkg: chkrootkit
```
Comment (mspanakis) [inline roles/monitoring/tasks/main.yml]: ola afta ta ksexwrista installation, mporeis na ta kaneis ena task me to with_items

File: roles/monitoring/tasks/main.yml [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+      src: chkrootkit.conf.j2 
+      dest: /etc/chkrootkit/chkrootkit.conf
+      owner: root
+      group: root
+      mode: 0644
+
+# Run Chkrootkit
+
+  - name: Run chkrootkit
+    shell: chkrootkit
```
Comment (mspanakis) [inline roles/monitoring/tasks/main.yml]: edw ti trexeis? An thes na xrisimopoiiseis to output tou, mporeis na to kaneis register...tsekare to 'register' kai pws mporeis na to xrisimopoiiseis...mporeis na kaneis px

mplamplampla
register: output_var

- debug: var=output_var gia na deis ti exei mesa afti i variable

Comment (mspanakis) [inline roles/monitoring/tasks/main.yml]: edw apla tha treksei mia fora...afto thes?

File: roles/monitoring/tasks/main.yml [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+
+# Search for open ports with nmap
+
+  - apt:
+      pkg: nmap
+      state: latest
+
+  - name: Scan for open ports
+    command: nmap -v -sT '{{ IP }}'
```
Comment (mspanakis) [inline roles/monitoring/tasks/main.yml]: kai pali mporeis na kaneis register to output gia na to xrisimopoiiseis somehow....

File: roles/monitoring/tasks/main.yml [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+
+  - name: SYN scanning
+    command: nmap -v -sS '{{ IP }}'
+
+# Use netstat to view open services
+ 
+  - name: Install netstat from apt
+    apt:
```
Comment (mspanakis) [inline roles/monitoring/tasks/main.yml]: ola afta ta apt mporeis na ta enwseis se ena...genika ola ta idia commands, vres na ta kaneis group giati einai pio efkolo na ta diavaseis ola mazemena

File: roles/monitoring/tasks/main.yml [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+# Use netstat to view open services
+ 
+  - name: Install netstat from apt
+    apt: 
+      name: net-tools 
+      state: present
+
+  - name: Open processes using netstat
+    shell: netstat -tunlp
+    register: results
```
Comment (mspanakis) [inline roles/monitoring/tasks/main.yml]: na afto elega me to register

File: roles/setup/tasks/main.yml [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+# tasks file for Setup
+
+  - name: Install pip from apt
+    apt: 
+      name: python-pip
+      state: present
+
+  - name: Update APT package cache
+    apt: 
+      update_cache: yes
```
Comment (mspanakis) [inline roles/setup/tasks/main.yml]: edw px tha mporouses na to enwseis me to apo panw...kai genika, prwta tha theleis na kaneis update to package cache kai meta na kaneis install pragmata gia na pareis tin latest version

File: roles/firewall/tests/test.yml (hunk 3 above)
Comment (XeniaSp) [reply roles/firewall/tests/test.yml:5]: den to katalava auto

Comment (mspanakis) [reply roles/firewall/tests/test.yml:5]: tpt eixe ena kokkino eikonidio sto telos pou elege "no new line at the end of file"

File: roles/monitoring/tasks/main.yml (hunk 10 above)
Comment (XeniaSp) [inline roles/monitoring/tasks/main.yml]: me voleuei nomizw kalutera na ta exw xwrista to kathe ena. Installation kai to config apo katw.. An omws thewreis oti einai kalytera etsi to allazw

File: roles/setup/tasks/main.yml (hunk 15 above)
Comment (XeniaSp) [reply roles/setup/tasks/main.yml]: auto dld pisteueis prepei na mpei sto prwto role pou trexei etsi?

**Pull request closed without merging**